docs/user-documentation/metadata.md

```diff
@@ -1,83 +1,130 @@
 
-!!! tip "Titles"
-    The included title field is limited to 255 characters; if your content has longer titles it is encouraged to create a separate long_title field to store the full title and reserve the default title field for a display title.
+![Screenshot of the "Manage fields" page for the "Repository Item" content type from islandora_defaults.](../assets/metadata_content_type_screenshot.png)
+
+!!! tip "Titles aren't fields."
+    Note that the "Title" field does not appear in this list. It is a built-in part of every content type. You can edit the label of this "field" if you want it called something other than "Title" under the "Edit" tab for that content type. This built-in title "field" is limited to 255 characters; if your content has longer titles it is encouraged to create a separate long_title field to store the full title and reserve the default title field for a display title. There is a contributed module called [Node Title Length](https://www.drupal.org/project/title_length), which allows an administrator to configure the length of the title field in the core node table. However, this only works on nodes (not media or other entities) and involves meddling in a core Drupal database schema, which makes some people uneasy.
 
 !!! tip "7.x Migration Note: What about my XML?"
-    In 7.x, metadata were (usually) stored within XML datastreams such as MODS or DC. In Islandora 8 we are breaking out
-    individual metadata elements into fields instead of using an attached XML document. The Metadata Interest Group is developing a default mapping which will provide a basic, yet customizable, transform between MODS metadata and Drupal fields in Islandora Demo.
+    In 7.x, metadata were usually stored using an XML schema such as MODS or DC, as datastreams attached to an object. In Islandora 8, metadata is stored as fields.
+    This means we are breaking out individual elements from a hierarchical structure to being individual independent values. Where some hierarchy or field grouping
+    is necessary, this can be done in Drupal using [Paragraphs](https://www.drupal.org/project/paragraphs), a widely-used Drupal contrib module. At the moment
+    (Nov 2019) we are working on the technical challenge of mapping data from paragraphs into RDF in Fedora.
+    The Metadata Interest Group has developed a default mapping ([spreadsheet](https://docs.google.com/spreadsheets/d/18u2qFJ014IIxlVpM3JXfDEFccwBZcoFsjbBGpvL0jJI/edit#gid=0), [guidance document](https://docs.google.com/document/d/15qSO9YcALtYSqd6CUuGx0t8FwUJ5pPwVPz0PA5rU898/edit?ts=5c5852f3#)) which provides a basic, yet customizable, transform between MODS metadata and Drupal fields in Islandora Defaults. It is suggested that individual institutions customize the mapping to meet their unique needs.
+
+    That said, if keeping the "legacy" XML metadata from 7.x is important to you, it can be attached to an Islandora 8 resource node as a Media entity.
```

Comment:
When I preview this page, this paragraph is broken out into a box with a horizontal scroll. I can't figure out where this is coming from based on the source text, but figured it was worth mentioning.

Reply:
Yeah, that's really weird. It's only in Github, because the `!!! tip` code is mkdocs-specific. It renders correctly in Mkdocs. The existing docs do the same thing.  https://github.com/Islandora/documentation/blob/master/docs/user-documentation/metadata.md 
![Screen Shot 2019-11-29 at 1 05 03 PM](https://user-images.githubusercontent.com/1943338/69883114-19368d80-12a9-11ea-9ee9-843387445d2b.png)